.github/workflows/csound_builds.yml

```diff
@@ -445,15 +443,13 @@ jobs:
           iscc /o. installer\\windows\\csound7_x64_github.iss
 
       - name: Upload installer
-        if: github.ref == 'refs/heads/develop'
```

Comment:
Just FYI, I added these conditions because we were getting storage/data usage warnings from Github. If we generate installers and zips for every branch, it adds up quickly.